src/components/Search.vue

```diff
@@ -18,8 +21,11 @@ export default {
   props: ['value', 'placeholder'],
   methods: {
     handleInput(e) {
-      this.$emit('input', e.target.value);
+      this.$emit('update:value', e.target.value);
```

Comment:
I would prefer to keep 'input' emit name instead of 'update:value'. I use input name in others components, i'd like to use the same naming everywhere.